### ISSUE TITLE
[UI] Warn for others

### ISSUE THREAD

**Issue by jucktnich** [corona-warn-app/cwa-app-ios#3492]
## Avoid duplicates
* [x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is specific for iOS only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [x] Bug is not already reported in another issue

## Technical details

- Device name: iPhone SE 1 (probably the device responsible for half of the UI bugs 😉😅)
- iOS version: iOS 14.7.1
- App version: 2.9.0 (4)

## Describe the bug

Text and pictures overlapping:
![image](https://user-images.githubusercontent.com/73582096/132532827-dec17e0c-b938-4584-be3d-ab61e391b102.MOV)

## Possible Fix

Remove or reconstraint background image if there's not enough space

**Comment by ndegendogo** [comment]
I see the same effect on both my devices
iPhone 8 / iOS 14.4.1 / cwa 2.10.1 (9)
iPhone 6 / iOS 12.5.3 / cwa 2.10.1 (0)

So this time not related to the special screen size of SE1 😅😅

To me it looks like the table cells have transparent background (this is the default), and in most cases it is fine to see the background of the view shining through
only in this case it looks a bit weird 😅

**Comment by dsarkar** [comment]
Good evening @jucktnich and @ndegendogo Thanks for reporting: Internal Tracking ID: [EXPOSUREAPP-9660](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9660)

Good night, DS

**Comment by ndegendogo** [comment]
It seems that the layout for the background with the image does not take the explanation text into account, so the image is positioned too high.
The explanation text needs more lines on devices with less width. You see this effect already on iPhone 8 if you compare with the iPhone 13 variants.

**Comment by ndegendogo** [comment]
**iPhone 8**

![Simulator Screen Shot - iPhone 8 - 2021-10-09 at 15 31 38](https://user-images.githubusercontent.com/6118267/136659868-26810604-efc3-4602-9ff9-0276bd38c7eb.png)

**Comment by ndegendogo** [comment]
**iPhone 13 Pro Max:**

![Simulator Screen Shot - iPhone 13 Pro Max - 2021-10-09 at 15 31 46](https://user-images.githubusercontent.com/6118267/136659959-7f549039-40a1-4762-8ded-314e0aaa98c7.png)

**Comment by Ein-Tim** [comment]
@jucktnich When you have time, could you check if version 2.13 has fixed this issue here?

**Comment by jucktnich** [comment]
I cannot confirm a fix. The picture is really small now, but still behind the text.

**Comment by Ein-Tim** [comment]
@jucktnich Could you share another screenshot with us? Thank you!

**Comment by jucktnich** [comment]
For sure
![image](https://user-images.githubusercontent.com/73582096/140184953-db72555f-8607-40c7-8bbc-1ad086f42bd2.png)
English version (its fixed there):
![image](https://user-images.githubusercontent.com/73582096/140184993-e0c6239a-47f7-4f1b-a431-eb691652de16.png)

**Comment by ndegendogo** [comment]
@jucktnich yes. This was the best I could achieve with the given text. In German it is even worse because the text is longer in that case.

Actually, I think the description text at the top is much too long. It should be shortened. 
See also #3709 what happens when you resize the font, even on a large screen.

**Comment by Ein-Tim** [comment]
@ndegendogo 

I thought the same and am on it to improve the text (shorten it).

**Comment by ndegendogo** [comment]
@jucktnich and, looking again at the layout: a bit improvement is maybe possible. The empty space between image and 'No events available', and above the 'Continue' button could be a bit smaller ...

**Comment by ndegendogo** [comment]
@Ein-Tim Thanks. Actually, the text is repeating much of what was already shown in the info screen before.

**Comment by Ein-Tim** [comment]
I wonder whether the word "erneut" ("Oder stellen Sie gelöschte Events wieder her, indem Sie den QR-Code des Events **erneut** einscannen.") resp. "again" ("Select an event for which you want to warn others on behalf of a person who has been diagnosed with coronavirus, or restore a deleted event by scanning the QR code of that event **again**.") is needed here. The QR code is not necessarily scanned again, as the event owner did probably never scan the QR cod before. 

Removing "erneut" would improve the situation at least a bit:
<img src="https://user-images.githubusercontent.com/67682506/140236352-93155079-df33-4b67-ae74-4132af96246a.png" width="300">

If we now also remove the "andere" from the description text below the image, everything is fixed:
<img src="https://user-images.githubusercontent.com/67682506/140236431-f383a48c-9b6f-4c33-81fb-fa27f30efb08.png" width="300">

However, we should keep in mind that this does not affect the functionality of the app in any way, so before making compromises in easy to understand texts, we should rather leave it as it is. 
Also, even if we may get this fixed in German & English, other supported languages are still affected by this...

**Comment by Ein-Tim** [comment]
One more comment: 
In general I think it would be better to simplify this screen by
1. Removing the text above the "QR-Code scannen" button and placing it all below the image and
2. Remove information which is not important or was already shown in the screen before. 

I will open an enhancement request on this later.

**Comment by Ein-Tim** [comment]
@jucktnich Would you maybe close this issue in favor of https://github.com/corona-warn-app/cwa-app-ios/issues/3791?

**Comment by jucktnich** [comment]
@Ein-Tim thx for the ping

**Comment by dsarkar** [comment]
@Ein-Tim @jucktnich OK, I closed the internal ticket, and mention that this os done in favour of https://github.com/corona-warn-app/cwa-app-ios/issues/3791.
